components/admin/invites/invites-page.tsx

```
@@ -83,21 +109,22 @@ export function InvitesPage() {
+		const phoneValidationError = validatePhone(newInvitePhone);
+		if (phoneValidationError) {
+			setPhoneError(phoneValidationError);
+			return;
+		}
 
 		setError(null);
 		const result = await createInvite({
 			email: emailResult.data,
 			name: newInviteName.trim(),
-			phone: newInvitePhone.trim(),
+			phone: phoneResult.data,
```

Biome (2.3.13)

[error] The phoneResult variable is undeclared.

By default, Biome recognizes browser and Node.js globals.
You can ignore more globals using the javascript.globals configuration.

(lint/correctness/noUndeclaredVariables)